packages/client/src/components/MovieComponent.js

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { Card, CardMedia, Typography, Rating, Box } from '@mui/material';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+
+const BasicRating = () => {
+  return (
+    <Rating
+      name="read-only"
+      value={4}
+      readOnly
+      sx={{
+        position: 'relative',
+        left: '2.6rem',
+        radius: '0.8',
+        height: '2.6rem',
+        width: '3rem',
+        top: '4.5rem',
+        color: '#003E2F',
+      }}
+    />
+  );
+};
+
+export default function MovieComponent() {
```

Comment:
I am unsure about your naming conventions in the group, but normally I would suggest more desciptive names in terms of business logic rather than programming logic. For example it is self-explananatory that this is a "component", but maybe there is better business name to call it? `MovieCard` or `MoveDetails` something like that?

Reply:
Ok, I'll change the component name, thank you!